src/nni_manager/core/sqlDatabase.ts

```diff
@@ -215,17 +219,21 @@ class SqlDB implements Database {
     public queryMetricData(trialJobId?: string, metricType?: MetricType): Promise<MetricDataRecord[]> {
         let sql: string = '';
         let args: any[] | undefined;
+        let jobId, parameterId: string | undefined;
+        if (trialJobId) {
+            [jobId, parameterId] = trialJobId.split("-");
+        }
         if (metricType === undefined && trialJobId === undefined) {
             sql = 'select * from MetricData';
         } else if (trialJobId === undefined) {
             sql = 'select * from MetricData where type=?';
             args = [metricType];
         } else if (metricType === undefined) {
-            sql = 'select * from MetricData where trialJobId=?';
-            args = [trialJobId];
+            sql = 'select * from MetricData where trialJobId=? and parameterId=?';
+            args = [jobId, parameterId];
```

Comment:
`parameterId` is in data field, how to query `parameterId`?

Reply:
MetricData table has this field also.

# | timestamp | trialJobId | parameterId | type | sequence | data
-- | -- | -- | -- | -- | -- | --
1 | 1588846040502 | fWpr7 | 2 | PERIODICAL | 0 | "97.41"